### PR TITLE
AQ-29233 Update Time-Series and Samples service models to 23.1

### PR DESCRIPTION
**Pull request by josef-mixan-ai**

### CODE REVIEW

File: src/main/java/com/aquaticinformatics/aquarius/sdk/timeseries/servicemodels/Acquisition.java [hunk 1]
```diff
@@ -823,6 +823,6 @@ public static class PostVisitFileBase implements IFileUploadRequest
 
     public static class Current
     {
-        public static final AquariusServerVersion Version = AquariusServerVersion.Create("22.3.75.0");
+        public static final AquariusServerVersion Version = AquariusServerVersion.Create("23.1.61.0");
```
Comment (ChrisKanaganayagam-AI) [inline src/main/java/com/aquaticinformatics/aquarius/sdk/timeseries/servicemodels/Acquisition.java:826]: This doesn't need to be the 23.1 GA version right? Just checking :)

**Review by ChrisKanaganayagam-AI — APPROVED** [review]

**Review by ChrisKanaganayagam-AI — APPROVED** [review]
Crazy how much more verbose the JAVA SDK is haha.

File: src/main/java/com/aquaticinformatics/aquarius/sdk/timeseries/servicemodels/Acquisition.java (hunk 1 above)
Comment (josef-mixan-ai) [reply src/main/java/com/aquaticinformatics/aquarius/sdk/timeseries/servicemodels/Acquisition.java:826]: No. It needs to be whatever version past which there were no API changes until the GA release.
Though if we need to make some changes say, addressing a bug in regression sprint that changes the API signature. We'd need to do this process again and publish new version of SDK, not that big deal.

File: src/main/java/com/aquaticinformatics/aquarius/sdk/timeseries/servicemodels/Provisioning.java [hunk 2]
```diff
@@ -270,427 +251,1464 @@ public static class DeleteLocationFolder implements IReturnVoid
...
+        /**
+        * Unique ID of the location
+        */
+        @ApiMember(DataType="string", Description="Unique ID of the location", Format="guid", IsRequired=true, ParameterType="path")
+        public String LocationUniqueId = null;
+
+        /**
+        * Standard identifier
+        */
+        @ApiMember(Description="Standard identifier", IsRequired=true)
```
[the remark this answers is not in the document]
Comment (josef-mixan-ai) [reply src/main/java/com/aquaticinformatics/aquarius/sdk/timeseries/servicemodels/Provisioning.java:1099]: All of it is code generated :)

**Pull request merged**